### PR TITLE
Add radix tabs component

### PR DESCRIPTION
**Pull request by TrySound**
Ref https://github.com/webstudio-is/webstudio-builder/pull/1516

- https://www.radix-ui.com/docs/primitives/components/tabs
- https://ui.shadcn.com/docs/components/tabs

## Code Review

- [ ] hi @kof, I need you to do
  - conceptual review (architecture, feature-correctness)
- [ ] hi @istarkov, I need you to do
  - detailed review (read every line)
  - test it on preview

## Before requesting a review

- [ ] made a self-review
- [ ] added inline comments where things may be not obvious (the "why", not "what")

## Before merging

- [ ] tested locally and on preview environment (preview dev login: 5de6)
- [ ] updated [test cases](https://github.com/webstudio-is/webstudio-builder/blob/main/apps/builder/docs/test-cases.md) document
- [ ] added tests
- [ ] if any new env variables are added, added them to `.env.example` and the `builder/env-check.js` if mandatory

### CODE REVIEW

File: packages/sdk-components-react-radix/src/tabs.ws.ts [hunk 1]
```diff
@@ -0,0 +1,197 @@
...
+  type: "container",
+  label: "Tabs List",
+  icon: RadioCheckedIcon,
+  requiredAncestors: ["Tabs"],
+};
+
+export const metaTabsTrigger: WsComponentMeta = {
+  category: "hidden",
+  type: "container",
+  label: "Tabs Trigger",
```
Comment (kof) [inline packages/sdk-components-react-radix/src/tabs.ws.ts:153]: Tab Trigger

File: packages/sdk-components-react-radix/src/tabs.ws.ts [hunk 2]
```diff
@@ -0,0 +1,197 @@
...
+  icon: RadioCheckedIcon,
+  requiredAncestors: ["TabsList"],
+  invalidAncestors: ["TabsTrigger"],
```
Comment (kof) [inline packages/sdk-components-react-radix/src/tabs.ws.ts:156]: TabTrigger

**Comment by kof** [comment]
Having Value as number on Tabs component and not having it anywhere else nor being able to select from a list is gonna be quite hard to understand, what can we do about it?


Screenshot 2023-08-02 at 16 36 42

**Comment by kof** [comment]
- [ ] changing the value on trigger and content and then deleting the value seems to not restore the original functionality

File: packages/sdk-components-react-radix/src/tabs.ws.ts (hunk 2 above)
Comment (TrySound) [reply packages/sdk-components-react-radix/src/tabs.ws.ts:156]: Tabs is a feature namespace. Called the same in radix. Consistent with all other components.

**Comment by TrySound** [comment]
> Having Value as number on Tabs component and not having it anywhere else nor being able to select from a list is gonna be quite hard to understand, what can we do about it?

So you don't want index to be internal thing?

**Comment by kof** [comment]
> So you don't want index to be internal thing?

The problem is that UI doesn't explain to user what to do. How will one know what index to set? Ideally its a select box, not an input

**Comment by TrySound** [comment]
We don't have a way to use select. This would require some querying dsl.

**Comment by TrySound** [comment]
What we could do is to have prop extension. This way we can show fallback when user clear the input and not loose the instance id when user change the value.

```
{
  type: 'string',
  value: '',
  fallback: {
    type: 'index',
    withinInstanceId: '...'
  }
}
```

**Review by istarkov — APPROVED** [review]

**Pull request merged**